css/main.css

```diff
@@ -5,133 +5,138 @@
     color: #1b1b1b;
 }
 
-a {
+.no-underline {
     text-decoration: none;
 }
 
 header {
-    text-align: left;
-    padding-inline: 10%;
+    position: relative;
+    padding: 0 max(50px, 10%);
     outline: 3px solid rgba(100, 100, 100, 0.459);
 }
 
-
 nav {
     height: 10vh;
+    max-height: 80px;
     line-height: 10vh;
     white-space: nowrap;
+    margin-left: 46%;
 }
 
 .upper-nav-bar {
     display: flex;
     align-items: center;
     justify-content: left;
     flex-direction: row;
+    list-style: none;
 }
 
-.upper-nav-bar > li:first-child{
-    margin-left: calc( 50% - 3em );
+.upper-nav-bar li {
+    padding: 0 3%;
 }
 
-.upper-nav-bar > li > a {
+.upper-nav-bar a {
     margin: auto;
-    padding-inline: 1em;
-    color:whitesmoke;
+    color: whitesmoke;
 }
 
 .first-screen {
     display: flex;
-    height: 70vh;
-    background: rgb(36, 31, 56);
-    padding-inline: 20%;
+    flex-direction: column;
     justify-content: center;
     text-align: center;
-    margin-top:-10vh;
+    height: 70vh;
+    min-height: 350px;
+    background: rgb(36, 31, 56);
+    padding: 0 20%;
```

Comment:
`px` for padding. Because you don't know which device client uses. So the layout can have differences with the design.